ada_feeding_perception/launch/ada_feeding_perception.launch.py

```diff
@@ -145,6 +145,21 @@ def generate_launch_description():
     )
     launch_description.add_action(face_detection)
 
+    # Load the table detection node
+    table_detection_remappings = [
+        ("~/table_detection", "/table_detection"),
+        ("~/toggle_table_detection", "/toggle_table_detection"),
+    ]
+    table_detection = Node(
+        package="ada_feeding_perception",
+        name="table_detection",
+        executable="table_detection",
+        remappings=realsense_remappings
+        + food_detection_remappings
```

Comment:
Why does it need `food_detection_remappings`?

Reply:
Looking at the code, I understand functionally why it is necessary. However, the variable name is confusing. Perhaps rename `food_detection_remappings` to `aligned_depth_remapping`, because fundamentally that is the only info it contains?